packages/web/app/src/components/ui/changelog/generated-changelog.ts

```diff
@@ -0,0 +1,26 @@
+export const latestChangelog = [
```

Comment:
Why do we commit that? It's exactly like types being generated by codegen, a thing that is part of the build process.

Reply:
I changed it to see what's exactly changed in changelogs, and added to prettier ignore

Reply:
Please add it to .gitignore as well

Reply:
Will do, just question I wonder if it's possible to have race condition? e.g. `app` is built before `docs` was built and generated a changelog?